datafusion/sql/src/planner.rs

```diff
@@ -446,7 +444,7 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
             for cte in with.cte_tables {
                 // A `WITH` block can't use the same name more than once
-                let cte_name = normalize_ident(&cte.alias.name);
+                let cte_name = normalize_ident(cte.alias.name.clone());
```

Comment:
previously normalize_indent *always* cloned. Now it only clones in a few places and most of the time can reuse the String in the sqlparser-ast directly